### DISCUSSION TITLE
Documentation for Middleware

### DISCUSSION THREAD

**Discussion by Kaitohooi** [dynamoose/dynamoose#1663] [Q&A]
Can I have the documentation of Dynamoose middleware? I didn't find it in the version 4 doc. Is that available in Dynamoose? I saw a ticket that said it was available. [https://github.com/dynamoose/dynamoose/issues/244](url)

**Comment by fishcharlie** [comment]
Middleware isn't supported in versions after v2.

Also. Please don't create duplicate comments. It's unnecessary and just ends up sending me duplicate emails. Thanks!

**Reply by Kaitohooi** [reply]
Sorry for that, is that any other workarounds for this issue?

**Reply by fishcharlie** [reply]
@Kaitohooi Nope. PRs are welcome.
